parsers/src/gltf/gltf_parser.cpp

```diff
@@ -465,7 +466,7 @@ rcbe::geometry::Mesh::FacetStorage getValuesByAccessor(
 
 namespace rdmn::parse::gltf {
 
-std::vector<rcbe::core::CoreObject> parse(
+R_PUBLIC_API std::vector<rcbe::core::CoreObject> parse(
```

Comment:
Strictly speaking it's not required, but it's a minor thing, can be removed in the future.